src/storage/src/cell_based_row_deserializer.rs

```diff
@@ -88,7 +116,7 @@ impl CellBasedRowDeserializer {
     pub fn take(&mut self) -> Option<(Vec<u8>, Row)> {
         let cur_pk_bytes = self.pk_bytes.take();
         cur_pk_bytes.map(|bytes| {
-            let ret = self.data.iter_mut().map(Option::take).collect::<Vec<_>>();
+            let ret = std::mem::replace(&mut self.data, vec![None; self.columns.len()]);
```

Comment:
Will this lead to extra allocation?

Reply:
I believe no. Previously, `.collect_vec()` will lead to one allocation. Now only `vec![None; self.columns.len()]` has one allocation.

Reply:
Make sense.